### PR TITLE
PMM-7295 Check for Supported DB's Explain/Example for QAN via UI tests

### PR DESCRIPTION
**Pull request by IvanaBrecherova**
[PMM-7295](https://jira.percona.com/browse/PMM-7295) [&#10088;?&#10089;](https://pullrequestbadge.com/?utm_medium=github&utm_source=percona&utm_campaign=badge_info)

### CODE REVIEW

**Review by atymchuk — APPROVED** [review]

**Comment by atymchuk** [comment]
@IvanaBrecherova Since there is a cla issue, pls see if you can get it solved. If not, please assign it to me once it's ready to be merged.

File: pmm-app/tests/QAN/pages/qanDetailsFragment.js [hunk 1]
```diff
@@ -28,6 +31,20 @@ module.exports = {
...
+  checkExamplesTab() {
+    I.waitForVisible(this.getTabLocator('Example'), 30);
+    I.click(this.getTabLocator('Example'));
+    I.wait(5);
+    I.dontSeeElement(this.elements.noExamples);
+  },
+
+  checkExplainTab(){
+    I.click(this.getTabLocator('Explain'));
+    I.wait(5);
```
Comment (puneet0191) [inline pmm-app/tests/QAN/pages/qanDetailsFragment.js:43]: No other way to handle this?

**Review by puneet0191 — APPROVED** [review]

File: pmm-app/tests/QAN/pages/qanDetailsFragment.js (hunk 1 above)
Comment (IvanaBrecherova) [reply pmm-app/tests/QAN/pages/qanDetailsFragment.js:43]: I will think about it again. :)

Comment (IvanaBrecherova) [reply pmm-app/tests/QAN/pages/qanDetailsFragment.js:44]: I was able to find pattern just for the Examples tab but not for Explain. So I deleted the wait in `checkExamplesTab()`

**Review by puneet0191 — APPROVED** [review]

**Pull request merged**